enterprise/cmd/worker/internal/permissions/bitbucket_projects.go

```diff
@@ -382,13 +382,13 @@ func newBitbucketProjectPermissionsResetter(db edb.EnterpriseDB, cfg *config, me
 			RecordResets:        metrics.resets,
 		},
 	}
-	return dbworker.NewResetter(workerStore, options)
+	return dbworker.NewResetter(logger.Scoped("dbworker.Resetter", ""), workerStore, options)
```

Comment:
```suggestion
	return dbworker.NewResetter(logger, workerStore, options)
```
The given logger is already scoped to `BitbucketProjectPermissionsResetter`